nodes/stampzilla-knx/tunnel.go

```diff
@@ -53,6 +53,18 @@ func newTunnel(node *node.Node) *tunnel {
 	}
 }
 
+func (tunnel *tunnel) GroupNames() []string {
+	ret := make([]string, len(tunnel.Groups))
```

Comment:
Isn't it a potential bug to read `tunnel.Groups` before locking?

Reply:
Yeah you are right. I thought len was safe... Can fix that!

Reply:
Fixed in #277